EX5 - Static Files Server/server.js

```diff
@@ -1,16 +1,34 @@
 const express = require('express');
 const fs = require('fs');
+const port = 3000; // Port to listen to
 
 let app = express();
+app.use(express.static('./files'));
 
+// Serve files from custom url of format: https://server/files/fileName (Defined it like this - didn't know if there is a special requirement).
+app.get('/server/files/:fileName', (req, res) => {
+	let fileName = req.params.fileName;
+	
+	if (!fileName){
+		// fileName passed is Null
+		 return res.send("No fileName param passed!");
+	}
+	
+	// Hardcoded path: server/files/...
+	let filePath = "./files/" + fileName;
```

Comment:
A security point: what if I put '..' as the file name? ;) think about it.

Reply:
Didn't think about it :| I'll look for a way to prevent it.
Thanks :)!